dev/Pager/PrototypePager/PrototypePager.Events.cs

```diff
@@ -0,0 +1,179 @@
+﻿using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Text;
+using Windows.UI;
+using Windows.UI.Notifications;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation.Provider;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace MUXControlsTestApp
+{
+    public sealed partial class PrototypePager : Control
+    { 
+        private static void OnSelectedIndexChanged(DependencyObject sender, DependencyPropertyChangedEventArgs args)
+        {
+            (sender as PrototypePager).PreviousPageIndex = (int)args.OldValue - 1;
+        }
+
+        private void OnPagerDisplayModeChanged()
+        {
+            switch (this.PagerDisplayMode)
+            {
+                case PagerDisplayModes.NumberBox:
+                    VisualStateManager.GoToState(this, NumberBoxVisibleVisualState, false);
+                    break;
+                case PagerDisplayModes.Auto:
+                case PagerDisplayModes.ComboBox:
+                    VisualStateManager.GoToState(this, ComboBoxVisibleVisualState, false);
+                    break;
+                case PagerDisplayModes.NumberPanel:
+                    VisualStateManager.GoToState(this, NumberPanelVisibleVisualState, false);
+                    break;
+            }
+        }
+
+        private void OnLastPageButtonVisibilityChanged()
+        {
+
+            switch (this.LastPageButtonVisibility)
+            {
+                case ButtonVisibilityMode.Auto:
+                case ButtonVisibilityMode.AlwaysVisible:
+                    VisualStateManager.GoToState(this, "LastPageButtonVisible", false);
```

Comment:
>LastPageButtonVisible [](start = 56, length = 21)

These names are also shared, can we make this a constant too

Reply:
Done!